dbt_project.yml

```
@@ -50,7 +50,7 @@ seeds:
 models:
   spellbook:
     +pre-hook:
-      - sql: "{{ set_trino_session_property(is_materialized(model) and is_partitioned(model), 'join_distribution_type', 'PARTITIONED') }}"
+      - sql: "{{ set_trino_session_property(is_join_distribution_enforced(model), 'join_distribution_type', model.config.get('join_distribution_type', 'AUTOMATIC')) }}"
```

We don't need a fallback as we never use it.
```suggestion
      - sql: "{{ set_trino_session_property(is_join_distribution_enforced(model), 'join_distribution_type', model.config.get('join_distribution_type')) }}"
```